### ISSUE TITLE
DateTime fields are not editable if data format is TOML

### ISSUE THREAD

**Issue by xelivous** [sveltia/sveltia-cms#228]
It seems like when editing entries the UI is hiding any field named "date" by default even if it's in the collection list. It will show up in the preview pane on the right but not in the edit pane. This happens with both a top level key and a nested key. More specifically the fields are visible when "creating" but not when "editing".

Example post:
```markdown
+++
title = "Test"
date = 2024-10-18T00:00:00

[nested_test]
date = 2024-10-17T00:00:00
+++
```

Example collection:
```yml
collections:
  - name: "posts"
    label: "Posts"
    folder: "content/posts"
    format: "toml-frontmatter"
    extension: "md"
    fields:
      - label: "Title"
        name: "title"
        widget: "string"
      - label: "Date"
        name: "date"
        widget: "datetime"
      - label: "Nested"
        name: "nested_test"
        widget: "object"
        fields:
          - label: "Nested Date"
            name: "date"
            widget: "datetime"
```

Example of a the nested date field missing in the edit pane on the left, but showing above "price" in the preview pane:
![2024-10-18_08-27-42](https://github.com/user-attachments/assets/047b7a7b-70eb-46ed-ad8d-8c3d4aee0805)

There's also strange behaviour with the summary field: `summary: "{{fields.date}}"` works fine but shows the massive datetime string, yet `summary: "{{fields.date | date('L')}}"` will simply be blank where the date is when the object is collapsed.

---

I can see the basic intention of this functionality, in certain SSG it is not best practice to edit the "date" field since it's interpreted as a "created_date" and you should instead change the "updated_at" field instead, but having this enforced with no way to opt out is a little strange; especially when it affects all fields with the name including nested fields.

If someone wants to replicate this functionality there should simply be a `hidden: true` for widgets, or maybe even a `disabled: true` to just toggle the input's "disabled" property and make it uneditable instead.

As far as I can tell the workaround is to just rename all of the fields to be something else (like `post_date` or `nested_test.nested_date`) but that's kind of strange.

**Comment by kyoshino** [comment]
Thanks for the heads-up! Will check.

**Comment by kyoshino** [comment]
Looks like nesting doesn’t matter; It’s due to a difference between the TOML parser and the JSON/YAML parsers. Fixinig.

**Comment by kyoshino** [comment]
Fixed in [v0.41.1](https://github.com/sveltia/sveltia-cms/releases/tag/v0.41.1) 🙇🏼

**Comment by xelivous** [comment]
Thanks for looking into it and the quick fix 👍
